docs/sphinx/dev_guide/kernel_class_impl.rst

```diff
@@ -43,28 +43,36 @@ The methods in the source file are:
       * Default problem size and number of kernel repetitions to generate 
         execution run time.
       * The actual problem size that will be run, which is a function of
-        the default size and command-line input.
-      * The number of *loop iterations* that are performed and the number of 
+        the default size and command-line input for target problem size.
+      * The number of *loop iterations* that are executed and the number of 
         loop kernels that run each time the kernel is executed. Note that the 
-        **ADD** kernel is based on a simple, single for-loop. However, other 
-        kernels in the Suite execute multiple loop kernels.
-      * The number of bytes read and written and the number of FLOPS performed 
-        for each kernel execution.
+        **ADD** kernel contains a single for-loop so the number of iterations 
+        is the problem size and number of kernels run is one. Other kernels in
+        the Suite execute multiple loop kernels with different sizes,
```

Comment:
The wording here is a little awkward and I'm not really sure what it's trying to say. Could this bullet point be reworded?

Reply:
I think I would take out "potentially" and say "Other kernels in the Suite may execute multiple loop kernels with different sizes, so these methods are used to describe these cases."

Reply:
Done.